### DISCUSSION TITLE
Why isn't `strategy: fixed` the default?

### DISCUSSION THREAD

**Discussion by tomdohnal** [floating-ui/floating-ui#1777] [General]
Hi, i'm just being curious, why isn't `strategy: fixed` the default setting?

To me it seems like you can accomplish everything when using `strategy: fixed` but can't with `strategy: absolute` (e. g. when using virtual elements) but it's likely that I'm missing something.

What kinda drawbacks can `strategy: fixed` have? (if any?) 

Thank uuuuu!!!

**Comment by FezVrasta** [comment]
Fixed requires the element position to be recomputed constantly as it's reference element container is scrolled.

**Comment by atomiks** [comment]
Another thing is how `position: fixed` acts during elastic overscrolling. In Chrome 105, it will not remain "anchored" during overscroll because `position: fixed` elements no longer move with the overscroll.

In Safari, this also happens (although in the opposite direction). However, this was a bug introduced in Safari 13 and looks like it will be fixed soon.

As for Chrome, it doesn't make `.scrollTop` change during elastic overscroll, unlike Safari, which is why it doesn't remain anchored. This is something the spec may add but for now it seems less reliable as a default than `absolute`.
